### PR TITLE
Add entity to NadeEvent

### PR DESCRIPTION
**Pull request by zyphie**

### CODE REVIEW

**Comment by markus-wa** [comment]
Sorry, I misunderstood your request. I thought this was about adding the thrower (`common.Player`) onto the event, which is already there.

I think either exposing `Parser.entities` via a new function `Parser.Entities()` would make more sense; otherwise this new `Entity` field would have to be added to all player related events to be consistent and it would be somewhat redundant.

Then you could use `Parser.Entities()[thrower.EntityID]`.

If that makes sense to you could you change the PR to that?

**Comment by markus-wa** [comment]
With `Parser.entities` exposed you could access the entities from anywhere else whenever you need them.

**Comment by markus-wa** [comment]
In fact, I just noticed `Player.Entity` already exists :sweat_smile:

Does this work for you?

**Comment by zyphie** [comment]
The entity I need is the entity of the nade, not the player. But you're probably right that it's better to have a generalized function to fetch entities, and then we can just add a field for entityID on the nade events. If you agree I can change the field etc to be for entityID instead.

**Comment by markus-wa** [comment]
Actually, in that case it makes sense, sorry.

Could you rename it to `NadeEntity` though?

Exposing `Parser.entities` is probably still a good idea though, feel free to do that as well.

File: parser.go [hunk 1]
```diff
@@ -79,6 +79,11 @@ func (p *Parser) GameState() *GameState {
 	return &p.gameState
 }
 
+// Entities returns the available entities
+func (p *Parser) Entities() *map[int]*st.Entity {
```
Comment (markus-wa) [inline parser.go]: This should return a `map[int]*st.Entity`
Go maps are already references by default

Comment (zyphie) [reply parser.go]: Ooops, my bad. Changed it.

**Comment by markus-wa** [comment]
Thanks for the contribution 🙌

**Pull request merged**